docs/tutorials/Usage/domain_tutorial.jl

```diff
@@ -52,7 +52,7 @@ but it is defined for all domains[^1].
 
 There is a single key method which take a ClimaLSM domain as an argument.
 
-- [` coordinates(domain)`](https://clima.github.io/ClimaLSM.jl/dev/APIs/SharedUtilities/#ClimaLSM.Domains.coordinates): under the hood, this function  uses
+- [` coordinates(domain)`](https://clima.github.io/ClimaLSM.jl/dev/APIs/sharedutilities/#ClimaLSM.Domains.coordinates): under the hood, this function  uses
```

Comment:
many of these links are not working in the generated docs... not sure why, not seeing an obvious mistake.

Reply:
sharedutilities, lowercase

Reply:
the link itself does work if you type it into the web browser, so somehow just in this PR something changed about the path?

Reply:
maybe! I guess it can be a small fix after we merge this, in case the path is a bit different.

Reply:
that's fine with me. ill add it to the docs overhaul issue

Reply:
I think it was because it is shared_utilities.